cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1199,7 +1207,11 @@ private boolean tryPullTimeSeriesSchemas(Node node, PullSchemaRequest request) {
       if (partitionGroup.contains(metaGroupMember.getThisNode())) {
         // this node is a member of the group, perform a local query after synchronizing with the
         // leader
-        metaGroupMember.getLocalDataMember(partitionGroup.getHeader()).syncLeader();
+        try {
+          metaGroupMember.getLocalDataMember(partitionGroup.getHeader()).syncLeader(null);
```

Comment:
maybe a dummy checkconsistent is better than null.

Reply:
If the value is null, the program skips it.
If it is a virtual method, the program needs to enter it.
This increases the stack call of the program jvm.

Reply:
ok